infra/aws/cloudformation/iam/cdn.packages.k8s.io/cloudformation.yaml

```diff
@@ -0,0 +1,210 @@
+Resources:
+  #########################################
+  # Terraform State                       #
+  #########################################
+
+  # TFStateBucket (packages-k8s-io-tfstate) is S3 bucket for storing Terraform
+  # state
+  TFStateBucket:
+    Type: AWS::S3::Bucket
+    # We retain the bucket to ensure that accidental deletion of the stack
+    # doesn't destroy Terraform state
+    DeletionPolicy: Retain
+    Properties:
+      BucketName: cdn-packages-k8s-io-tfstate
+      AccessControl: Private
+
+  #########################################
+  # IAM user for Terraform                #
+  #########################################
+
+  # ProvisionerUser (provisioner) is an IAM user used for running Terraform
+  ProvisionerUser:
+    Type: 'AWS::IAM::User'
+    Properties:
+      Path: /
+      UserName: provisioner
+      Policies:
+        - PolicyName: ProvisionerUserAccess
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Action: 'sts:AssumeRole'
+                Effect: Allow
+                Resource: !Sub 'arn:aws:iam::${AWS::AccountId}:role/Provisioner'
+
+  # ProvisionerRole is an IAM role assumed by provisioner IAM user to run Terraform
+  ProvisionerRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      RoleName: Provisioner
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Action: 'sts:AssumeRole'
+            Effect: Allow
+            Principal:
+              AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:user/provisioner'
+    DependsOn: ProvisionerUser
+
+  # ProvisionerPlanAccessPolicy is an IAM policy that grants permissions to run `terraform plan`
+  ProvisionerPlanAccessPolicy:
+    Type: 'AWS::IAM::ManagedPolicy'
+    Properties:
+      ManagedPolicyName: ProvisionerPlanAccess
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: 'Allow'
+            Action:
+              - 'acm:DescribeCertificate'
+              - 'acm:ListTagsForCertificate'
+            Resource: !Sub 'arn:aws:acm:us-east-1:${AWS::AccountId}:certificate/*'
```

Comment:
Why the hard coded region here? I'd add a comment to explain the thinking.

Reply:
There's a comment in Terraform configs that I can copy over. The tl;dr version is that CloudFront requires ACM certificates to be located in the `us-east-1` region.